validate_url_service.py

```diff
@@ -0,0 +1,12 @@
+from typing import List
+
+from base_db_api import BaseDBAPI
+
+
+allDBs: List[BaseDBAPI]
```

Comment:
You still need to initialize is with `[]` or `None` or whatever. Anyway I really don't like globals